src/internal.c

```diff
@@ -13217,12 +13217,20 @@ int wolfSSH_oct2dec(WOLFSSH* ssh, byte* oct, word32 octSz)
 }
 
 
+/* not using UINT_MAX here because word32 should always be 32 bits where an
+ * int type can change depending on platform */
+#define WS_WORD32_MAX_SZ 4294967295U
```

Comment:
Is it too ugly to use something like `#define WS_WORD32_MAX_SZ (word32)((1ULL << (sizeof(word32)*4)) - 1)`?

Reply:
Something like that would work. I get the idea, doing a bitshift to larger than the max size minus 1. Will make the change.

Reply:
Looking for WOLFSSL_BIT_SIZE (8) I see we have WOLFSSL_MAX_32BIT (0xffffffU), which is what I tried to find initially. Will try to use that instead of recreating it.